searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/UnzipFile.java

```diff
@@ -57,6 +67,10 @@ private void unzip(Path path) throws UnzipFileException {
         }
 
     }
+    /**
+     *
```

Comment:
Empty documentation. Is it necessary?

Reply:
it's empty because i still need to had a meeting with the person who do this class